searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -0,0 +1,138 @@
+package org.jalau.at18.searchobject.model.facedetector;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
+
+
+import org.jalau.at18.searchobject.common.logger.At18Logger;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfRect;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.objdetect.CascadeClassifier;
+
+import java.util.logging.Logger;
+
+
+/**
+ * It is responsible for analyze an image and detect if there it's a face or not.
+ *
+ * @author Daniela Martinez
+ * @version 1.0
+ */
+public class FaceDetect {
+    private String faceDetection;
+    private static String current_dir; //full image path direction
```

Comment:
try using the naming conventions, change `current_dir` to `currentDir`

Reply:
already change it